recipes/asio-grpc/all/test_v1_package/test_package.cpp

```diff
@@ -0,0 +1,36 @@
+#ifdef ASIO_GRPC_V2
```

Comment:
Please, move .cpp and .proto to test_package. When Conan 2.0 be declared as default, test_v1_package will be deleted.